Packs/SymantecBlueCoatProxySG/ReleaseNotes/1_0_5.md

```diff
@@ -0,0 +1,7 @@
+#### Modeling Rules
+##### Symantec BlueCoat ProxySG Modeling Rules
+- Updated the Symantec BlueCoat ProxySG Modeling Rules to include mapping for fields:
+  - xdm.target.ipv4
+  - xdm.target.ipv6
+
+- Modified the log format which appears in the pack's readme, to include r_ip field.
```

Comment:
Modified the log format to include the r_ip field, which appears in the content pack's ReadMe.